gulpfile.js

```diff
@@ -62,12 +62,23 @@ gulp.task('watch', function () {
 /**
  * Validate source JavaScript
  */
-gulp.task('jshint', function () {
+gulp.task('lint', function () {
   return gulp.src(lintFiles)
     .pipe(plumber())
-    .pipe(jshint())
-    .pipe(jshint.reporter('jshint-stylish'))
-    .pipe(jshint.reporter('fail'));
+    .pipe(eslint({
+      extends: 'eslint:recommended',
```

Comment:
I just inlined the configuration options here, but I saw that crosslead-platform has a `.eslintrc` file. I suggest—in the PR—a way to avoid falling into this adhocity trap.